ppx/mel_ast_invariant.ml

```diff
@@ -30,34 +30,40 @@ module Warnings = struct
     | Fragile_external of string
     | Redundant_mel_string
     | Deprecated_non_namespaced_attribute
+    | Uncurried_arity0
 
   let kind = function
     | Unused_attribute _ -> "unused"
     | Fragile_external _ -> "fragile"
     | Redundant_mel_string -> "redundant"
     | Deprecated_non_namespaced_attribute -> "deprecated"
+    | Uncurried_arity0 -> "uncurried"
 
   let pp fmt t =
     match t with
     | Unused_attribute s ->
         Format.fprintf fmt
           "Unused attribute [%@%s]@\n\
            This means such annotation is not annotated properly.@\n\
-           For example, some annotations are only meaningful in externals\n"
-          s
+           For example, some annotations are only meaningful in externals" s
     | Fragile_external s ->
         Format.fprintf fmt
           "%s : the external name is inferred from val name is unsafe from \
            refactoring when changing value name"
           s
     | Redundant_mel_string ->
         Format.fprintf fmt
-          "[@mel.string] is redundant here, you can safely remove it"
+          "`[@mel.string]' is redundant here, you can safely remove it"
```

Comment:
Why the weird quotes? (I think i've seen them in some OCaml console msgs, but that doesn't make them less weird 😄

Reply:
They're used in OCaml, and IIRC dune. I just wanted to minimize the surprise factor